project/AutomaticModuleName.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
+ */
+package play
+
+import sbt.Def
+import sbt._
+import sbt.Keys._
+
+/**
+ * Helper to set Automatic-Module-Name in projects.
+ *
+ * !! DO NOT BE TEMPTED INTO AUTOMATICALLY DERIVING THE NAMES FROM PROJECT NAMES !!
+ *
+ * The names carry a lot of implications and DO NOT have to always align 1:1 with the group ids or package names,
+ * though there should be of course a strong relationship between them.
+ */
+object AutomaticModuleName {
```

Comment:
Hum, if we are adding this to multiple projects in playframework org, maybe we should add it to playframework/interplay instead?

Reply:
I don't know scala or the playframework build, so from the example I found it put it in this location. It will probably be needed for all play and scala libraries so if their is a better location I'm happy to raise that PR.

Reply:
@nhojpatrick what @marcospereira is referring to is https://github.com/playframework/interplay/tree/master/src/main/scala/interplay

Reply:
@marcospereira @ignasi35 see https://github.com/playframework/interplay/pull/111